src/templater.sh

```diff
@@ -48,6 +48,7 @@ function templater() {
 
   # cat "${templateDst}"
 
+  # shellcheck disable=SC2016
   if grep '${' "${templateDst}" && [ "${ALLOW_REMAINDERS}" = "0" ]; then
```

Comment:
that's true but this PR is not about our style guide but only about shellcheck which is why i didn't cleanup everything.